docs/specs/xref.yml

```diff
@@ -321,3 +321,36 @@ outputs:
   docs/a.json:
   docs/b.json: |
     { "content": "<p>Link to <a href=\"/docs/a\">Title from yaml header a</a>\nLink to <a href=\"/docs/a\">Title from yaml header a</a>\nLink to <a href=\"/docs/a\">Title from yaml header a</a>\nLink to <a href=\"/docs/a\">Title from yaml header a</a>\nLink to <a href=\"/docs/a\">Title from yaml header a</a></p>" }
+---
+# Refer to uid within link
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    ---
+    title: Title from yaml header a
+    fullName: full name
+    uid: a
+    ---
+  docs/b.md: | 
+    Link to [title](xref:a?displayProperty=fullName#bookmark)
+outputs:
+  docs/a.json:
+  docs/b.json: |
+    { "content": "<p>Link to <a href=\"/docs/a#bookmark\">title</a></p>" }
```

Comment:
>/docs/a#bookmark [](start = 39, length = 16)

resolve to relative path for intermal xref?

Reply:
I will fix this in another PR